runtime/verutil/cfrerr.c

```diff
@@ -87,6 +125,12 @@ getJ9CfrErrorDetailMessageNoMethod(J9PortLibrary* portLib, J9CfrError* error, co
 	case J9NLS_CFR_ERR_BAD_BOOTSTRAP_ARGUMENT_ENTRY__ID:
 		errorString = getJ9CfrErrorBsmMessage(portLib, error, className, classNameLength);
 		break;
+	case J9NLS_CFR_ERR_MAJOR_VERSION__ID:
```

Comment:
This can use the new ID `J9NLS_CFR_ERR_MAJOR_VERSION2__ID`. The code in cfreader.c can set the new ID instead of `J9NLS_CFR_ERR_MAJOR_VERSION__ID`. The text of the unused old message `J9NLS_CFR_ERR_MAJOR_VERSION` can be removed from the cferr*.nls files. Same for MINOR.

Reply:
done.